src/appleseed.python/bindaov.cpp

```diff
@@ -91,6 +94,21 @@ namespace
 
         return channels;
     }
+
+    Image& get_crypto_image(AOV* aov)
+    {
+        const string model = aov->get_model();
+
+        if (model == "cryptomatte_object_aov" || model == "cryptomatte_material_aov")
+            return static_cast<CryptomatteAOV*>(aov)->get_crypto_image();
+
+        PyErr_SetString(PyExc_RuntimeError, "AOV is not a Cryptomatte AOV");
+        bpy::throw_error_already_set();
+
+        // If AOV is not a Cryptomatte type, return a blank pointer for the image.
+        Image* dummy = nullptr;
```

Comment:
This is going to crash since you're dereferencing a null pointer.

If `get_crypto_image()` must be able to fail, it could return a pointer instead of a reference.

Reply:
I’ll have to see how to do that with Boost Python. Right now it fails to compile info try to return a pointer

Reply:
Never mind, guess it works fine.